receiver/k8seventsreceiver/README.md

```diff
@@ -21,13 +21,16 @@ the K8s API server. This can be one of `none` (for no auth), `serviceAccount`
 - `namespaces` (default = `all`): An array of `namespaces` to collect events from.
 This receiver will continuously watch all the `namespaces` mentioned in the array for
 new events.
+- `initial_lookback` (default = `60s`): Allow the events having timestamp
```

Comment:
Why not `1m`? :-)

Reply:
In the proposed [design doc](https://docs.google.com/document/d/1M9M6alW0ftmsBPBftKtH4h3eewpq263Ep678qU0YlB4/edit?usp=sharing), we had a discussion on the same and concluded to use seconds. I think it's general practice that is being used, for ex.: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/statsdreceiver/factory.go#L35

Reply:
Would you mind sharing more information about it here? Skimming through the doc, I couldn't find the reasoning for it.

Reply:
Previously, I got one [suggestion](https://docs.google.com/document/d/1M9M6alW0ftmsBPBftKtH4h3eewpq263Ep678qU0YlB4/edit?disco=AAAAOxFUWao) on the design doc to use `60s` rather than `1m`. Also, I found a few instances(for ex. [awscontainerinsightreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/factory.go#L33), [googlecloudspannerreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/googlecloudspannerreceiver/factory.go#L31), and [prometheusexecreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusexecreceiver/factory.go#L34)) in this repo where `60s` is being used so I have also followed the same practice over here.

Reply:
That's fine, I was just curious why 60s is better than 1m.